TestSwiftyDropbox/TestSwiftyDropbox_iOS/AppDelegate.swift

```diff
@@ -38,33 +37,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                     print("Error: \(description)")
                 }
             }
+            let mainController = self.window!.rootViewController as! ViewController
+            mainController.checkButtons()
```

Comment:
This force unwrap is completely unnecessary. Your not using it as a non-optional parameter anywhere.

Reply:
you are correct, will fix.